scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1482,9 +1482,9 @@ boolean changeStartAt(JobId jobId, String startAt)
 
     /**
      * @param jobId job id of existing job
-     * @return copy of the object which was submitted to the scheduler
+     * @return copy of the xml which was submitted to the scheduler
      */
-    Job getJobContent(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException,
+    String getJobContent(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

Reply:
I think sonarqube means that there is one exception in the throws declaration which is not actually thrown, can you check that?